### PR TITLE
Vinu/94687/add info in reports

### PR DESCRIPTION
**Pull request by vinu-deriv**

### CODE REVIEW

File: src/components/statements.jsx [hunk 1]
```diff
@@ -18,16 +21,43 @@ const Statements = () => {
...
         setStatements(transactions);
         setStatementCount(count);
       });
     }
   });
 
+  const getDisplayName = (symbol, activeSymbols) => {
+    if (symbol && activeSymbols.length) {
+      const required_symbol = activeSymbols.find((active_symbol) => {
+        return active_symbol.symbol === symbol;
```
Comment (nada-deriv) [inline src/components/statements.jsx]: `const required_symbol = activeSymbols.find(active_symbol => active_symbol.symbol === symbol)`

File: src/components/statements.jsx [hunk 2]
```diff
@@ -18,16 +21,43 @@ const Statements = () => {
     if (active_account) {
       sendRequest({
         statement: 1,
+        description: 1,
         limit: 100,
         offset: 0,
       }).then((resp) => {
         const { transactions, count } = resp.statement;
+        transactions.forEach((transaction) => {
```
Comment (nada-deriv) [inline src/components/statements.jsx:26]: `transactions.forEach((transaction) => {` or
`const { transactions = [], count = 0 } = resp.statement;`

**Review by nijil-deriv — APPROVED** [review]

File: src/components/statements.jsx [hunk 3]
```diff
@@ -7,8 +7,11 @@ import classNames from "classnames";
 import { Show } from "solid-js";
 import { For } from "solid-js";
```
Comment (amina-deriv) [inline src/components/statements.jsx]: cant we do these in a single statement?

File: src/components/statements.jsx [hunk 4]
```diff
@@ -7,8 +7,11 @@ import classNames from "classnames";
...
 import { addComma } from "../utils/format-value";
+import { getMarketInformation } from "../utils/map-markets";
 import { login_information } from "../stores/base-store";
 import Loader from "./loader";
+import { activeSymbols } from "../stores";
+import { getContractConfig, isHighLow } from "../utils/contract";
```
Comment (amina-deriv) [inline src/components/statements.jsx:6]: can we reorder these import statements?

Comment (vinu-deriv) [reply src/components/statements.jsx:6]: made changes

**Review by nada-deriv — APPROVED** [review]

**Review by amina-deriv — APPROVED** [review]

**Pull request merged**